app/src/test/java/org/fossasia/openevent/general/AppLinkUtilsTest.kt

```diff
@@ -0,0 +1,55 @@
+package org.fossasia.openevent.general
+
+import android.net.Uri
+import org.fossasia.openevent.general.utils.EVENT_IDENTIFIER
+import org.fossasia.openevent.general.utils.RESET_PASSWORD_TOKEN
+import org.fossasia.openevent.general.utils.AppLinkUtils
+import org.fossasia.openevent.general.utils.VERIFICATION_TOKEN
+import org.junit.Test
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+private const val EVENT = "event"
+private const val RESET_PASSWORD = "resetPassword"
+private const val VERIFY_EMAIL = "verifyEmail"
+
+@RunWith(RobolectricTestRunner::class)
+class AppLinkUtilsTest {
+
+    private fun getAppLink(type: String): Uri {
+        return when (type) {
+            EVENT -> Uri.parse("https://eventyay.com/e/5f6d3feb")
+            RESET_PASSWORD -> Uri.parse("https://eventyay.com/reset-password?token=822980340478781748445098077144")
+            VERIFY_EMAIL -> Uri.parse("https://eventyay.com/verify?token=WyJsaXZlLmhhcnNoaXRAaG")
```

Comment:
I'm don't quite understand the hard-coded URL in here. So every user will reset password/verify email with the token above? Does it matter?

Reply:
Nope, these are for the unit tests under test implementation

Reply:
oh yeah sorry, didn't take a closer look